packages/chart/src/ChartUtils.ts

```diff
@@ -1963,7 +1962,7 @@ class ChartUtils {
       scene: {
         xaxis: this.makeLayoutAxis(dh.plot.AxisType.X, theme),
         yaxis: this.makeLayoutAxis(dh.plot.AxisType.Y, theme),
-        zaxis: this.makeLayoutAxis(dh.plot.AxisType.Z, theme),
+        zaxis: this.makeLayoutAxis(null, theme),
```

Comment:
Ah, on Enterprise it did used to have this constant. I guess we still need to set this for theming purposes of 3d plots in plotly-express? Wondering if we can just remove it and `makeLayoutAxis` doesn't need to take `null` then.

Reply:
We use just the X or Y in `makeLayoutAxis` to add a few params to X or Y axis specifically

That said, w/ the theme changes I think that could be removed entirely and in the defaults here we could instead do something like this for the 1-2 props we add to the X and Y axis. I'd say do that as a followup though to keep this PR just related to types

```ts
xaxix: {
  ...this.makeLayoutAxis(theme),
  showgrid: true
}
```